### DISCUSSION TITLE
1.4.4

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#48] [Announcements]
1. 优化UI
2. 移除路由狗里的 LUCI 接口
3. 修复 docker 预览的一些问题
4. 相册增加上传按钮

**Comment by nanshancc** [comment]
可以加入docker compose吗？

**Reply by Carseason** [reply]
有没有相关的案例截图？

**Reply by nanshancc** [reply]
![屏幕截图 2024-08-07 133521](https://github.com/user-attachments/assets/e1ac646d-4a9a-43a5-b6f9-c4462edeca7e)

**Comment by no-duke** [comment]
没法上传图标，提示没有设置储存位置发自我的荣耀手机-------- 原始邮件 --------发件人： Carseason ***@***.***>日期： 2024年8月5日周一 傍晚6:00收件人： Carseason/routerdog ***@***.***>抄送： no-duke ***@***.***>, Comment ***@***.***>主    题： Re: [Carseason/routerdog] 1.4.4 (Discussion #48)
有没有相关的案例截图？

**Reply by Carseason** [reply]
你可以建个issues那发图
